.travis.yml

```
@@ -0,0 +1,30 @@
+language: go
+
+# SCOPE OF THIS :
+# we use travis only for running the acceptance tests for operators.
+# Circleci isolated container network so we cannot run kind in circleci.
+#
+# Add all other tests in circleci and use  make e2e-test
+services:
+        - docker
+
+go:
+  - '1.11.x'
+
+services:
+          - docker
+
+jobs:
+  include:
+    - stage: Integration Tests
+      before_script:
+        # Download and install kubectl
+        - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.12.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+        
+        # Download and install Kind
+        - curl -Lo kind https://github.com/kubernetes-sigs/kind/releases/download/0.0.1/kind-linux-amd64 && chmod +x kind && sudo mv kind /usr/local/bin/
+        
+        # Create a new Kubernetes cluster using KinD
+        - kind create cluster
+
+      script: make e2e-tests
```

I would move the previous setup for `kind` to its own Makefile target, so the tests lunch would be done with:
```suggestion
      script: make kind e2e-tests
```

---

adressed thx

---

This is marked as resolved, however the script still uses the target `make kind-e2e-tests`  not two targets `kind` and  `e2e-tests`

---

we have 2 targets, 1 doing the install before and creation of cluster and then this one which is calling the e2e-tests for kind , but we have also a generic call